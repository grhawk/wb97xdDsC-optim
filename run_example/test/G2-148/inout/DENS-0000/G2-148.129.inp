 $BASIS GBASIS=N31 NGAUSS=6 $END
 $DATA
 Should be setted!
 C1
 O     8.00     0.027191     1.363691    -0.167516
 C     6.00    -0.000926     0.036459     0.370128
 H     1.00     0.859465     1.775647     0.121307
 H     1.00     0.007371     0.082145     1.470506
 C     6.00    -1.313275    -0.563514    -0.088979
 C     6.00     1.200721    -0.764480    -0.104920
 H     1.00    -1.334005    -0.607253    -1.181009
 H     1.00     1.202843    -0.807817    -1.197189
 H     1.00    -2.147812     0.054993     0.247676
 H     1.00     2.136462    -0.299324     0.223164
 H     1.00    -1.438709    -1.574275     0.308340
 H     1.00     1.177736    -1.784436     0.289967
 $END
 $DFT NLEB=110 DDSC=.t. NRAD=24 $END
 $SYSTEM MWORDS=8 $END
 $SCF DIRSCF=.t. $END
 $CONTRL RUNTYP=ENERGY MAXIT=200 DFTTYP=wB97X ICHARG=0 MULT=1 
EXETYP=RUN SCFTYP=ROHF $END