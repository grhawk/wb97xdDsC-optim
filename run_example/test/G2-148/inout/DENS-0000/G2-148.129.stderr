/var/spool/slurm/job10166/slurm_script: line 9: module: command not found
